autoload/fz.vim

```diff
@@ -107,12 +107,19 @@ function! fz#run(...)
     return
   endif
 
+  " Get basepath
+  let basepath = get(ctx['options'], 'basepath', '')
+  if basepath != ''
+    let basepath = expand(basepath)
+  endif
+
   " check type
   let typ = get(ctx['options'], 'type', 'cmd')
   if typ == 'cmd'
     let $FZ_IGNORE = get(ctx['options'], 'ignore', '(^|[\/])(\.git|\.hg|\.svn|\.settings|\.gitkeep|target|bin|node_modules|\.idea|^vendor)$|\.(exe|so|dll|png|obj|o|idb|pdb)$')
     let fz_command = get(ctx['options'], 'fz_command', g:fz_command)
     let cmd = get(ctx['options'], 'cmd', g:fz_command_files)
+    let cmd = isdirectory(basepath) ? printf(cmd, basepath) : printf(cmd, '')
```

Comment:
```
let cmd = printf(cmd, basepath)
```

Reply:
Fumm, if given path is not valid, nothing to show.

Reply:
Fixed.